submissions/task_html-css-popup/style.css

```diff
@@ -0,0 +1,388 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* DECLARE .HEADER */
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    width: calc(100%-80px);
+    height: 50px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* HEADER - Hide dots in the <ul> element*/
+
+.menu__list {
+    list-style: none;
+}
+
+/* HEADER - Style for <a> element */
+
+a {
+    text-decoration-style: dashed;
+    text-decoration-color: white;
+    text-underline-offset: 3px;
+}
+
+a:focus {
+    outline: 3px dashed gold;
+    text-decoration: underline;
+}
+
+a:hover,
+a:active {
+    text-decoration: underline;
+    text-decoration-color: white;
+}
+
+/* DECLARE .MAIN */
+
+.main {
+    display: flex;
+    width: calc(100%-80px);
+    height: 75vh;

```

Comment:
If you need to stretch content in main on all height of window, you need to add this to parent el
````
display: flex;
flex-direction: column;
min-height: 100vh;
````

and this to main and remove height: 75vh;

````
flex: auto;
````

Reply:
I've added two classes .body & .main in CSS to resolve the issue. Thank you, Hanna, for your kind help 😄